docs/development/faq.md

```diff
@@ -26,3 +26,14 @@ If you get an `undefined: Asset` error it is likely because you did not run `mak
 First, in the Fleet UI, navigate to the App Settings page under Admin. Then, in the "SMTP Options" section, enter any email address in the "Sender Address" field, set the "SMTP Server" to `localhost` on port `1025`, and set "Authentication Type" to `None`. Note that you may use any active or inactive sender address.
 
 Visit [locahost:8025](http://localhost:8025) to view Mailhog's admin interface which will display all emails sent using the simulated mail server.
+
+## Adding fake hosts for testing
```

Comment:
I think it's a bit more clear to remove "fake" from this. What do you think? I'm open to it if you feel it enhances clarity.

Reply:
I agree. Now that I read it again, "fake" doesn't add any more clarity. "For testing" communicates the idea clearly on its own.